### ISSUE TITLE
Stream larger chunks of audio

### ISSUE THREAD

**Issue by adrianlyjak** [adrianlyjak/obsidian-tts#3]
The player splits text sentence-by-sentence, and renders those to speech. It does this so that content is streamable and navigable.

However, the speech synthesis isn't as good. 

Would it be possible [to stream the audio chunks from openai](https://platform.openai.com/docs/guides/text-to-speech), and somehow rethink the navigation chunks within the paragraph?

**Comment by Calorion** [comment]
Or at least preload future sentences in the background, to reduce loading times.

**Comment by adrianlyjak** [comment]
@Calorion the plugin currently buffers 3 sentences ahead. Were you seeing delays between sentences?

**Comment by Calorion** [comment]
Yes, occasionally, but if you're already buffering three sentences ahead, that's probably all you should do.
